src/Servers/IIS/IIS/src/Core/IISHttpContext.Log.cs

```diff
@@ -38,7 +38,9 @@ public static void UnexpectedError(ILogger logger, string className, Exception e
                 _unexpectedError(logger, className, methodName, ex);
             }
 
+#pragma warning disable CS0618 // Type or member is obsolete
             public static void ConnectionBadRequest(ILogger logger, string connectionId, BadHttpRequestException ex)
```

Comment:
Most of these usages can switch to referencing the base type either inline or with a using statement in the file. That will avoid excess pragmas. The two places we need to directly reference the old types are when it's first created & thrown, as well as tests that check which type was thrown. All of the other catch blocks and fields can use the new base type.